### PR TITLE
Added Django 1.11-2.2 test envs (and removed older ones) and fixes

### PR DESCRIPTION
**Pull request by dakrauth**
The upstream repo was moved over to github from bitbucket a few years ago, but it seemed easier (and within the estimated number of points) to quickly tweak these files to work with Django 2.0-2.2.

Also, the upstream uses `python-ldap` instead of `pyldap`, and I didn't want to invest the time to see what those ramifications are.

There is 1 failing test, and it is consistent across all environments: `django_auth_ldap.tests.LDAPTest.test_populate_with_attrlist`

This also seems to fail in older versions, so I'm fine with the consistency, unless someone who is much more familiar with LDAP protocols would like to investigate.

### CODE REVIEW

File: django_auth_ldap/backend.py [hunk 1]
```diff
@@ -793,7 +778,8 @@ def _mirror_groups(self):
             new_groups = [Group.objects.get_or_create(name=name)[0] for name
                           in target_group_names if name not in existing_group_names]
 
-            self._user.groups = existing_groups + new_groups
+            # self._user.groups = existing_groups + new_groups
```
Comment (defbyte) [inline django_auth_ldap/backend.py]: do we want to keep this commented-out line around?

File: setup.cfg [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [egg_info]
-tag_build = +atl.1.0.0
+tag_build = +atl.2.0.0
```
Comment (defbyte) [inline setup.cfg:2]: major version change given not backwards compatible 👍

**Review by defbyte — COMMENTED** [review]
just to double check, this version is not backwards compatible, yeah?

File: django_auth_ldap/backend.py (hunk 1 above)
Comment (dakrauth) [reply django_auth_ldap/backend.py]: No, I was supposed to clean that up

**Review by obswork — APPROVED** [review]

**Pull request merged**